prusti-launch/src/lib.rs

```diff
@@ -26,6 +34,12 @@ pub fn get_current_executable_dir() -> PathBuf {
         .to_path_buf()
 }
 
+pub fn get_prusti_contracts_dir(exe_dir: PathBuf) -> PathBuf {
+    let mut prusti_dir = exe_dir.parent().unwrap().parent().unwrap().to_path_buf();
+    prusti_dir.extend(["prusti-contracts", "target", "verify", "release"]);
```

Comment:
From seeing that the tests pass I guess so. The thing is that this will never be an issue with cargo builds (c.f. `get_prusti_contracts_dir` is only called in `if !cargo_invoked { ... }`) and is only relevant for raw `prusti-rustc`, where you can toggle optimisations with the `-O` flag. I doubt this would cause issues there though, since optimisations are only relevant after Prusti/early-MIR.